app/src/main/res/layout/fragment_installed_add_on_details.xml

```diff
@@ -29,21 +29,24 @@
                 android:checked="true"
                 android:clickable="true"
                 android:focusable="true"
+                android:layout_marginStart="54dp"
                 android:padding="16dp"
                 android:text="@string/mozac_feature_addons_settings_on"
-                android:textSize="18sp" />
+                android:textColor="?primaryText"
+                android:textSize="16sp" />
 
             <TextView
                 android:id="@+id/settings"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:layout_below="@+id/enable_switch"
                 android:background="?android:attr/selectableItemBackground"
-                android:drawablePadding="12dp"
+                android:drawablePadding="32dp"
                 android:padding="16dp"
                 android:text="@string/mozac_feature_addons_settings"
-                android:textSize="18sp"
-                app:drawableStartCompat="@drawable/mozac_ic_preferences"
+                android:textSize="16sp"
+                android:textColor="?primaryText"
+                app:drawableStartCompat="@drawable/ic_settings"
```

Comment:
Are `ic_settings` and `mozac_ic_preferences` different? Should we uplift?

Reply:
I'm not quite sure why, but they have different sizes than drawables in AC. I guess they want to have their own customization 🤷‍♂